src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -132,6 +147,9 @@ public final void refreshItems() {
 
         listView.restoreSelection();
         this.setId(guiController.getDefaultRepo() + "_col" + panelIndex);
+        totalLabel.setText(String.format("Total: %d ", issueCount));
+
```

Comment:
For the benefits of generalising with other statistics in the future, this line can be extracted into another method called `updateFooter` or `updateStatistics` for example. (`updateFooter` seems better to me in this case since it's more UI oriented).